packages/web/src/components/molecules/Console/Console.tsx

```diff
@@ -202,6 +202,29 @@ export const Console = forwardRef<ConsoleRef, ConsoleProps>(({content, wrap, Lin
   };
   useEventCallback('scroll', onScroll, containerRef?.current);
 
+  useEffect(() => {
```

Comment:
Could be [`useClientRect`](https://github.com/kubeshop/testkube-dashboard/blob/develop/packages/web/src/hooks/useClientRect.ts), [`useInViewport`](https://github.com/kubeshop/testkube-dashboard/blob/develop/packages/web/src/hooks/useInViewport.ts) or directly [`react-use/useIntersection`](https://github.com/streamich/react-use/blob/master/src/useIntersection.ts)

Reply:
Refactored